### ISSUE TITLE
segmentation accuracy of lung nodule on LUNA16

### ISSUE THREAD

**Issue by yjsb** [MrGiovanni/ModelsGenesis#57]
I crop lung nodule to 64*64*64 and turn the gray level to [0,255], and then turn it to [0,1] as the input of UNet3D. I used the pre-trained  weight excluded the final_layer, and I replaced it as a two channels output (for one-hot output). But I found that I can't get a better initial weight than my UNet3D trained using random initial. I wonder if the operation that I replace the final_layer is wrong. I hope that I can get your help.

**Comment by yjsb** [comment]
the UNet3D input is 64x64x64 around the nodule, as your experiments in UNet++.

**Comment by MrGiovanni** [comment]
Hi @yjsb 

We used 64x64x32 in Models Genesis because the model was pre-trained in such a way. 
A concrete example of nodule segmentation can be found at https://github.com/MrGiovanni/ModelsGenesis/blob/master/keras/downstream_tasks/lung%20nodule%20segmentation.ipynb

Thanks,

Zongwei
